### DISCUSSION TITLE
Issue with SFTP connection

### DISCUSSION THREAD

**Discussion by sidharthmohanty** [iterate-ch/cyberduck#14739] [Q&A] [answered]
Hi team,

I hope you are doing well!!

I just found an issue with the sftp connection. I'm using my s3 bucket and trying to connect to specific directory but I can see all of my other sibling directories. Is it possible for you hide the other directories apart from the actual one(for which I'm trying to connect with SFTP creds)

Thanks!!

**Comment by gmahe** [comment, marked as answer]
This has nothing to do with Cyberduck, this is about the permissions of that IAM user. If you are using transfer family you should have [scoped down policies](https://docs.aws.amazon.com/transfer/latest/userguide/logical-dir-mappings.html) to only allow users to see their root directory
